src/components/manage-identities/ManageIdentities.tsx

```diff
@@ -384,8 +388,8 @@ export function ManageIdentities() {
             </Flex>
             <Alert status="success" mt="2">
               <AlertDescription fontSize="13">
-                Somebody who can perform update / delete operations on{" "}
-                {"identity"} wallet.
+                User which can perform update / delete operations on{" "}
+                {"identity"} entity.
```

Comment:
{"identity"} => identity

Reply:
this is done automatically, because we have the variable with the same name